src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationRepositoryTests.cs

```diff
@@ -1046,6 +1048,8 @@ public async Task GetPublicationForUser_NoReleaseMethodologyLegacyRelease()
 
             var publication = new Publication
             {
+                Title = "Test title",
+                Summary = "Test summary",
```

Comment:
I've renamed the test.

I've not thought through what will happen after the further refactoring. My guess is the test will stick around in some shape or form, but perhaps `GetPublicationForUser` will return a `Publication` rather than a `PublicationViewModel` like has happened with the release repository methods in this PR.